spec/features/journeys_spec.rb

```diff
@@ -53,6 +68,10 @@ def sign_in(user)
     click_link 'Označiť ako vybavené'
 
     expect(page).to have_content('Vybavené!')
+
+    visit user_journeys_path
```

Comment:
Prosim nedavat do existujucich testov novu logiku, spravme na to separe test. Aj nizsie.

Reply:
V poriadku, vytiahnem to do zvlášť testov.

Reply:
Testy sú už oddelené.

Reply:
Ja to tu este stale vidim.

Reply:
Moja chyba, až teraz.